uktrade/data-hub-frontend
# Cls2 398 edit likelihood of landing

## Description of change

Added auto scroll when you edit an active investment from the company overview page. 

## Test instructions
Go to this page:
http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/overview

Scroll down and find the active investment projects section and click the edit link. The page should auto focus to the likelihood of landing field.


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
